crates/indexer/Cargo.toml

```
@@ -41,11 +41,12 @@ tokio = { version = "1.13.0", features = ["rt"] }
 tokio-amqp = "1.0.1"
 
 # Solana & Metaplex
-anchor-lang = "0.19.0"
+anchor-lang = "0.20.0"
 metaplex = "0.0.1"
 metaplex-auction = "0.0.1"
-metaplex-token-metadata = "0.0.1"
+mpl-token-metadata = { version="~1.2.4", features = [ "no-entrypoint" ] }
```

```suggestion
mpl-token-metadata = { version="~1.2.4", features = ["no-entrypoint"] }
```

Should we add `no-entrypoint` to the other MPL imports?

---

we should, metaplex uses it

---

@imabdulbasit do you wanna go ahead and fold that into this PR? seems like a good time to do it